css/css-sizing/aspect-ratio/block-aspect-ratio-040.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: The definite max-height should win the transferred minimum height</title>
+<link rel="author" title="Cathie Chen" href="mailto:cchen@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio-minimum">
```

Comment:
Change this reference to https://drafts.csswg.org/css-sizing-4/#aspect-ratio-size-transfers